app/src/main/res/layout/item_tab_contents.xml

```diff
@@ -1,40 +1,85 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:clipChildren="false"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:animateLayoutChanges="false"
-    android:background="?attr/paper_color"
-    android:orientation="vertical"
-    android:padding="16dp">
-
-    <TextView
-        android:id="@+id/tab_article_title"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_gravity="center_vertical"
-        android:fontFamily="serif"
-        android:textColor="?attr/primary_color"
-        android:textSize="24sp"
-        tools:text="Lorem ipsum" />
-
-    <TextView
-        android:id="@+id/tab_article_description"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_gravity="center_vertical"
-        android:layout_marginTop="8dp"
-        android:textColor="?attr/placeholder_color"
-        android:textSize="14sp"
-        tools:text="Lorem ipsum" />
-
-    <ImageView
+    android:layout_height="180dp">
+
+    <org.wikipedia.views.WikiCardView
+        android:id="@+id/tabCardView"
         android:layout_width="match_parent"
-        android:layout_height="300dp"
-        android:layout_marginTop="16dp"
-        android:scaleType="fitStart"
-        app:srcCompat="@drawable/ic_text_placeholder"
-        android:contentDescription="@null"/>
+        android:layout_height="match_parent"
+        android:layout_margin="4dp"
+        app:cardBackgroundColor="?attr/paper_color">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/tabContainer"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="?attr/selectableItemBackground">
+
+            <ImageView
+                android:id="@+id/tabCloseButton"
+                android:layout_width="24dp"
+                android:layout_height="24dp"
+                android:padding="4dp"
```

Comment:
Would it be possible to set the `width` and `height` to `48dp` with some extra padding setup in order to follow the accessibility guidelines?
```

                android:layout_width="48dp"
                android:layout_height="48dp"
                android:paddingBottom="26dp"
                android:paddingStart="26dp"
                android:paddingEnd="4dp"
                android:paddingTop="4dp"
```

Reply:
The current design is consistent with Google's own design in the Chrome app, but I do agree with your suggestion, with a slight tweak.